docs/architecture/adr-048-consensus-fees.md

```diff
@@ -0,0 +1,181 @@
+# ADR 048: Consensus Fees
```

Comment:
```suggestion
# ADR 048: Multi Tire Gas Price System
```

Reply:
Maybe we should find a better title? The ADR proposes a system for allowing tires in the block space based on the gas price.